lib/options/sender_option_parser.rb

```diff
@@ -93,6 +95,20 @@ def initialize(args)
         @options.msg_content = msg_content
       end
 
+      # Message property
+      @opt_parser.on(
+        "-P",
+        "--msg-property KEY=VALUE",
+        String,
+        "property specified as KEY=VALUE (use '~' instead of '=' for auto-casting)"
+      ) do |msg_content_map_item|
+        if @options.msg_properties.nil?
+          @options.msg_properties = {}
+        end
+
+        key, value = parse_kv(msg_content_map_item)
```

Comment:
Rename msg_content_map_item to msg_property.

Reply:
not here, here the name is correct